README.txt

```diff
@@ -0,0 +1,75 @@
+This webfont is generated by http://fontello.com open source project.
+
+
+================================================================================
+Please, note, that you should obey original font licenses, used to make this
+webfont pack. Details available in LICENSE.txt file.
+
+- Usually, it's enough to publish content of LICENSE.txt file somewhere on your
+  site in "About" section.
+
+- If your project is open-source, usually, it will be ok to make LICENSE.txt
+  file publicly available in your repository.
+
+- Fonts, used in Fontello, don't require a clickable link on your site.
+  But any kind of additional authors crediting is welcome.
```

Comment:
Jak już zamieszczamy licencje bibliotek zewnętrzych, zróbmy to w osobnym katalogu i ponazywajmy te pliki odpowiednio. A najlepszym rozwiązaniem jest utworzenie pliku `README.md` i w nim opisanie jakie biblioteki zewnętrzne zostały wykorzystane w projekcie i na jakiej licencji.